frontend/contexts/sessionProvider.tsx

```diff
@@ -106,6 +113,7 @@ export const SessionProvider: React.FC<{ children: ReactNode }> = ({
                     return "";
                 }
                 if (response.account_status === AccountStatus.PENDING) {
+                    pendingChecked = true; // needs to be false, otherwise the next request will think it's automatically true, even when it has an invalid key.
```

Comment:
> needs to be false

but you set it to true? hahahahah

Reply:
ahaha, my bad this was an old comment, it needed to be removed